about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="resources/icons/mycollection/font/flaticon.css"
+    />
+    <link rel="stylesheet" type="text/css" href="resources/styles-about.css" />
+    <script src="menu.js" defer></script>
+    <title>About</title>
+  </head>
+  <body>
+    <div class="first-line">
+      <img class="icon-facebook" src="resources/icons/facebook.svg" alt="facebook icon">
+      <img class="icon-twitter" src="resources/icons/twitter.svg" alt="twitter icon">
+      <p>English</p>
+      <p>My page</p>
+      <p>Logout</p>
+    </div>
+    <button type="button" class="button-menu">
+      <i class="flaticon-menu" id="button-icon"></i>
+    </button>
+    <nav class="topnav">
+      <a href="index.html"
+        ><img src="resources/images/logo-crypto-con-dark-grey.svg" alt="" />
+      </a>
+      <ul class="menu">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html#main-program-home">Program</a></li>
+        <li><a href="#btn-main-program-home">Join</a></li>
+        <li><a href="#partners-crypto-con">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#" class="cc-campaign">CC Crypto</a></li>
+      </ul>
```

Comment:
- [ ] I really liked your minimalist approach to the mobile menu. However, it's a project requirement to follow the original design provided for this template. Kindly the background color must be either transparent or the same color/pattern as the top section background.

![p3](https://user-images.githubusercontent.com/57562869/152657261-7124fcb2-f598-4100-9d98-714e316de524.png)

Reply:
Hi @moise10r! The design provided does not show any design for the mobile menu. For the color background of the menu, I use #ec5242 which is part of the color palette given in the template.